e2etest/bgptests/infra_setup.go

```diff
@@ -66,79 +71,106 @@ func InfraSetup(ipv4Addresses, ipv6Addresses []string, cs *clientset.Clientset)
 		See `e2etest/README.md` for more details.
 	*/
 
-	ibgpSingleHopContainerConfig := frrcontainer.Config{
-		Name: "ibgp-single-hop",
-		Neighbor: frrconfig.NeighborConfig{
-			ASN:      metalLBASN,
-			Password: "ibgp-test",
-			MultiHop: false,
+	frrContainers := []frrcontainer.Config{
```

Comment:
why the need to use a slice here? If it's only for convenience of using the slice in config.For, then create a local array.

Reply:
I reverted to the way it was before without a slice